### ISSUE TITLE
Concurrent SyncFlow and NormalizeFlow

### ISSUE THREAD

**Issue by iskakaushik** [PeerDB-io/peerdb#678]
When doing incremental syncs, currently we run `SyncFlow` and then run `NormalizeFlow`:

https://github.com/PeerDB-io/peerdb/blob/0fe793e3703f2ca38a5908518834702c05ade9ff/flow/workflows/cdc_flow.go#L334C1-L365C3

In reality `NormalizeFlow` doesn't depend or need to run after `SyncFlow`, running both one after another results in performance issues where if `NormalizeFlow` runs for longer than 3-5 minutes, which we have seen at larger deployments, the slot grows during this time. This issue is to make them run concurrently.

cc: @serprex

**Comment by serprex** [comment]
There's a dependency: https://github.com/PeerDB-io/peerdb/blob/4a72fea2570d541d7e8c258de04fe9a4da393d50/flow/workflows/cdc_flow.go#L369

Is the optimized behavior to have both run in parallel & have schema changes apply to future invocations?

**Comment by iskakaushik** [comment]
@serprex exactly, thats the ideal behavior.

**Comment by heavycrystal** [comment]
duplicate of #55

**Comment by heavycrystal** [comment]
closed via #893
